api/src/main/java/org/pmiops/workbench/firecloud/FireCloudServiceImpl.java

```diff
@@ -303,22 +295,17 @@ public void removeOwnerFromBillingProject(
       String ownerEmailToRemove, String projectName, Optional<String> callerAccessToken) {
     Preconditions.checkArgument(ownerEmailToRemove.contains("@"));
 
-    final BillingApi scopedBillingApi;
+    final BillingV2Api billingV2Api;
 
     if (callerAccessToken.isPresent()) {
-      // use a private instance of BillingApi instead of the provider
-      // b/c we don't want to modify its ApiClient globally
-      final ApiClient apiClient = firecloudApiClientFactory.newApiClient();
-      apiClient.setAccessToken(callerAccessToken.get());
-      scopedBillingApi = new BillingApi();
-      scopedBillingApi.setApiClient(apiClient);
```

Comment:
Why was this removed? It seems like the original intent was to allow overriding the access token, but now the access token override value is unused.

Reply:
My understanding of this code is if callerAccessToken is present that means the call is from user. In this case we should just use `endUserApi`. Otherwise, we use `serviceAccountApi`.
Probably this code was writing before we invite those two type of API provider.

Reply:
Interestingly, our only use of `removeOwnerFromBillingProject` passes `Optional.empty()`. I think it'd be best to remove the parameter and logic entirely since we don't need it.

Reply:
changed to `removeOwnerFromBillingProjectAsService`